app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= sanitize t("users.new.count_error.text",count: pluralize(@user.errors.size, t("error")))%>.
```

Comment:
sanitize để làm gì a ?

Reply:
Sanitize là method của SanitizeHelper dùng để :
- Loại bỏ các tag và attribute của HTML input trừ các phần tử an toàn đã biết.
- Loại bỏ các thuộc tính href/src với các giao thức không an toàn như javascript:, đồng thời chống lại các nỗ lực sử dụng các tham chiếu ký tự Unicode, ASCII và hex, xóa tất cả các ký tự đặc biệt.
- Sanitize mặc định là Rails :: Html :: WhiteListSanitizer.
- Có thể custom các roles của santize.
- Tuy nhiên sanitize cũng không được đảm bảo một cách tuyệt đối về tính an toàn và định dạng.